### DISCUSSION TITLE
LRV Bitrate Adjustment Not Possible on Hero 13?

### DISCUSSION THREAD

**Discussion by sjmarcy2** [gopro/labs#1171] [Q&A] [answered]
You can do it on earlier GoPros, but not the 13.  I use them for Data with Video uses such as this example where basically just thumbnails are used.  
<img width="1642" alt="Screenshot 9" src="https://github.com/user-attachments/assets/5874aeb7-3856-4e97-a885-dc4507f892c0">

**Comment by dnewman-gpsw** [comment, marked as answer]
Not currently possible. On older cameras the LRV used H.264, MP4s used HEVC. The newer cameras use HEVC for both (so the BITH is not going to help.)  Why do you want to change the bit-rate of the LRVs?  Do you want higher or lower bit-rate?

**Reply by sjmarcy2** [reply]
Higher bit rate.   Small files for use as data with video rather than video with data as an overlay such as the screenshot shows. Quicker to get the data from the camera via WiFi script.  And since LRVs have the GPS / IMU data they are useful.    Saves time with the high rez main files able to be used later on, if needed..  Often they are not necessary since the LRV is good enuf to see car position well enough to help the driver decide changes.     The latest GPS processor (11 & 13) seems to be quite decent!

**Reply by dnewman-gpsw** [reply]
So why higher bit-rate? If you are only using them for thumbnails and metadata, wouldn't lower be better for faster transfers over WiFi?

**Comment by sjmarcy2** [comment]
Because they look better at larger than the roughly 4 Mbps rate, yet are still v small compared to the primary files.    I did a comparison years ago, posted on the GoPro site with samples.   For the primary video files, I am loving the HLG option and those of course involve much larger files / rez.    It sounds like for the better LRV, it may be best to use an earlier camera, s'ok I have many.
